src/Program.cs

```diff
@@ -26,6 +31,20 @@ public static class Program {
                 .AddCommandLine(args)
                 .Build();
 
+            {
+                if (args.Any(a => a.ToLowerInvariant().Contains("wfd") || a.ToLowerInvariant().Contains("waitfordebugger")))
```

Comment:
I would put a #if DEBUG around it.   Just to make sure it is not enabled/working in a production release image and I think this is anyway only needed for debug, right?.